test.php

```diff
@@ -0,0 +1,7 @@
+<?php
+    $eeeeeeeeee;
+        $aaaaaaaaaa;
+    $bbbbbbbbbb;
+    $aaaaaaaaaa;
+
+?>
```

Comment:
__[staging][code_sniffer]__
* A closing tag is not permitted at the end of a PHP file

<!-- Comment by [SideCI](https://sideci.com) -->